apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -1079,8 +1080,15 @@ translation("prometheus") ->
         {"vm_system_info_collector_metrics", fun tr_vm_system_info_collector/1},
         {"vm_memory_collector_metrics", fun tr_vm_memory_collector/1},
         {"vm_msacc_collector_metrics", fun tr_vm_msacc_collector/1}
+    ];
+translation("vm_args") ->
+    [
+        {"+P", fun tr_vm_args_process_limit/1}
     ].
 
+tr_vm_args_process_limit(Conf) ->
+    2 * conf_get("node.max_ports", Conf, ?DEFAULT_MAX_PORTS).
```

Comment:
Maybe this should be _at least_ `2 * max_ports` rather than fixed at this number?

That is, we could specify a greater number of processes than `2 * max_ports` and have that?

Reply:
It doesn't seem like there are any scenarios that require such flexible settings.
Even if the user sets it to 2*maxports + 1, it will still become 4*maxports, so I don't think it's necessary to expose this parameter to users. 

Keep simple for user side.
http://erlang.org/documentation/doc-10.0/erts-10.0/doc/html/erl.html#+P

> NOTE: The actual maximum chosen may be much larger than the Number passed. Currently the runtime system often, but not always, chooses a value that is a power of 2. This might, however, be changed in the future. The actual value chosen can be checked by calling [erlang:system_info(process_limit)](http://erlang.org/documentation/doc-10.0/erts-10.0/doc/html/erlang.html#system_info_process_limit).